RxSwiftUtilities.xcodeproj/project.pbxproj

```diff
@@ -847,7 +873,7 @@
 				PRODUCT_NAME = RxSwiftUtilities;
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```

Comment:
No reason to raise the minimum to 5.0 as RxSwift 5 still supports Swift 4.2

Reply:
I think this changed automatically by XCode. I will revert it and check if it builds.